libvirt/resource_libvirt_pool.go

```diff
@@ -207,11 +254,14 @@ func resourceLibvirtPoolRead(ctx context.Context, d *schema.ResourceData, meta i
 		poolPath = poolDef.Target.Path
 	}
 
-	if poolPath == "" {
-		log.Printf("Pool %s has no path specified", pool.Name)
-	} else {
-		log.Printf("[DEBUG] Pool %s path: %s", pool.Name, poolPath)
-		d.Set("path", poolPath)
+	// for logical pool the path auto-generated, so we don't set/read it
```

Comment:
But it is missing reading the source devices. I am fixing that in my local branch which I will push here.